react-array-methods/src/model/known-methods.ts

```diff
@@ -1,29 +1,34 @@
-const knownMethods = () => [
+interface IMethodWithArgs {
+  method: string;
+  args: any[];
+}
+
+const knownMethods = (): IMethodWithArgs[] => [
   { method: 'concat', args: [['a', 'b', 'c']] },
   { method: 'copyWithin', args: [0, 3, 4] },
   { method: 'entries', args: [] },
-  { method: 'every', args: [(item) => item < 4] },
+  { method: 'every', args: [(item: number) => item < 4] },
   { method: 'fill', args: [0, 3, 4] },
-  { method: 'filter', args: [(item) => item < 4] },
-  { method: 'find', args: [(item) => item > 3] },
-  { method: 'findIndex', args: [(item) => item > 3] },
+  { method: 'filter', args: [(item: number) => item < 4] },
+  { method: 'find', args: [(item: number) => item > 3] },
+  { method: 'findIndex', args: [(item: number) => item > 3] },
   { method: 'flat', args: [] },
-  { method: 'flatMap', args: [(item) => [item, item * 2]] },
-  { method: 'forEach', args: [(item) => item * 2] },
+  { method: 'flatMap', args: [(item: number) => [item, item * 2]] },
+  { method: 'forEach', args: [(item: number) => item * 2] },
   { method: 'includes', args: [3] },
   { method: 'indexOf', args: [3] },
   { method: 'join', args: [] },
   { method: 'keys', args: [] },
   { method: 'lastIndexOf', args: [3] },
-  { method: 'map', args: [(item) => item * 2] },
+  { method: 'map', args: [(item: number) => item * 2] },
   { method: 'pop', args: [] },
   { method: 'push', args: [15] },
-  { method: 'reduce', args: [(acc, value) => acc + value] },
-  { method: 'reduceRight', args: [(acc, value) => acc + value] },
+  { method: 'reduce', args: [(acc: number, value: number) => acc + value] },
+  { method: 'reduceRight', args: [(acc: number, value: number) => acc + value] },
```

Comment:
may be it's an a time to separate aargis in 3 or 4 consts?

Reply:
Done